docs/C-Data-Structures/arrays.md

```diff
@@ -1,20 +1,24 @@
 ---
+id: arrays
+title: Arrays
 sidebar_position: 1
+slug: /data-structures/arrays
+description: An array is a data structure consisting of an ordered set of elements of common type that are stored contiguously in memory.
 ---
 # Arrays
 
 ## Learning Outcomes
 
 After reading this section, you will be able to:
 
-* Design data collections using arrays to manage information efficiently
-* Introduce character strings as terminated collections of byte information
+- Design data collections using arrays to manage information efficiently
+- Introduce character strings as terminated collections of byte information
 
 ## Introduction
 
-Programs can process extremely large amounts of data much faster than well-established manual techniques.  Whether this processing is efficient or not depends in large part on how that data is organized.  For example, large collections of data can be organized in structures if each variable shares the same type with all other variables and the variables are stored contiguously in memory.  Not only can structured data be processed efficiently but the programming of tasks performed on structured data can be simplified considerably.  Instead of coding a separate instruction for each variable, we code the instruction that is common to all variables and apply that instruction in an iteration across the data structure.
+Programs can process extremely large amounts of data much faster than well-established manual techniques.  Whether this processing is efficient or not depends in large part on how that data is organized.  For example, large collections of data can be organized in [structures](../C-Data-Structures/structures "Structures") if each variable shares the same type with all other variables and the variables are stored contiguously in memory.  Not only can structured data be processed efficiently but the programming of tasks performed on structured data can be simplified considerably.  Instead of coding a separate instruction for each variable, we code the instruction that is common to all variables and apply that instruction in an iteration across the data structure.
```

Comment:
Please check, this link gives an error.

Reply:
Thanks! I've updated the intra-site links.